layouts/index.html

```diff
@@ -7,9 +7,9 @@
     <a name="page-top"></a>
     {{ partial "nav.html" . }}
 
-    {{ with $.Site.Data.index }}
+    {{ with .Resources.ByType "hero" }}
       {{ range . }}
```

Comment:
Due to us using Resources we can just do {{ range .Resources }}. This removes the need for the above with.